frappe/website/doctype/web_form/web_form.py

```diff
@@ -692,6 +692,13 @@ def get_link_options(web_form_name, doctype, allow_read_on_all_link_options=Fals
 			if value and int(value) == 1:
 				show_title_field_in_link = True
 
+		if not title_field:
```

Comment:
@akhilnarang this is needlessly complicated!

Just use `frappe.get_meta(doctype)` which is mix of doctype + property setter.

Reply:
![image](https://github.com/frappe/frappe/assets/9079960/3824f18d-8c5d-426f-a4c4-2370c08cefb2)

Reply:
Thanks, will open a PR to simplify

Reply:
#23360